src/Core/Usecase/UpdateUsecase.php

```diff
@@ -69,9 +69,13 @@ public function isSearch()
 	// Usecase
 	public function interact()
 	{
+    //remove created time stamp if present to prevent overwriting
+    if ($this->payload['created']) {
```

Comment:
I don't think this should happen in the Usecase.. there are 2 places we could handle it:
1. If you add `created` to the immutable fields on the Entity that would work.. however it might not work for `updated` since that needs to change
2. Drop/Overwrite the created and updated values in the repository class.. I think thats probably the best place for now.